apps/fees-pay/automation/kustomization.yaml

```diff
@@ -67,3 +67,6 @@ resources:
   - ../refund-notifications-job/image-repo.yaml
   - ../refund-notifications-job/image-policy.yaml
   - ../refund-notifications-job/demo-image-policy.yaml
+  - ../unprocessed-payment-update/image-repo.yaml
```

Comment:
Hey - I think you still want to add the perftest/ithc images here so they're deployed

Reply:
Added ithc and perftest image